### PR TITLE
eo: add API for querying the class type

### PR DESCRIPTION
**Pull request by marcelhollerbach**
a class can be a interface mixin abstract or regular.
This adds a API for getting this information

### CODE REVIEW

File: src/lib/eo/Eo.h [hunk 1]
```diff
@@ -2003,6 +2004,8 @@ EAPI Eina_Value efl_property_reflection_get(Eo *obj, const char *property_name);
 
 #include "efl_class.eo.h"
 
+EAPI Efl_Class_Type efl_class_type_get(const Efl_Class *klass);
```
Comment (Bluebugs) [inline src/lib/eo/Eo.h]: Someone has to complain about the lack of documentation ;-)

**Comment by zmike** [comment]
Shouldn't the commit message be "querying" and not "quering"

File: src/lib/eo/Eo.h (hunk 1 above)
Comment (marcelhollerbach) [reply src/lib/eo/Eo.h]: Added docs :)

**Comment by segfaultxavi** [comment]
Change the commit title as zmike said and you'll get my seal of approval.

**Pull request merged**